lib/raygun.batch.ts

```diff
@@ -52,24 +54,13 @@ export class RaygunBatchTransport {
    * @return Promise with response or error if rejected
    */
   send(options: SendOptions): Promise<IncomingMessage> {
-    return new Promise((resolve, reject) => {
-      this.onIncomingMessage({
-        serializedMessage: options.message,
-        // TODO: Switch to using Promises internally
-        // See issue: https://github.com/MindscapeHQ/raygun4node/issues/199
-        callback: (error, message) => {
-          if (error) {
-            reject(error);
-          } else if (message) {
-            resolve(message);
-          }
-        },
-      });
+    const promise = this.onIncomingMessage(options.message);
 
-      if (!this.timerId) {
-        this.timerId = setTimeout(() => this.processBatch(), 1000);
-      }
-    });
+    if (!this.timerId) {
```

Comment:
We don't need `=== undefined` because this is not changed anywhere else and setTimeout will never return 0, or false, right?

Reply:
It is indeed initialized to null here: https://github.com/MindscapeHQ/raygun4node/blob/7d9a3cf677757ee6fe7e14eb3794e8f256bf6cee/lib/raygun.batch.ts#L36C11-L36C18

This PR did not change how the `timerId` works.